src/vit-testing/registration-service/src/client/args.rs

```diff
@@ -72,29 +67,13 @@ impl JobCommand {
                 Ok(())
             }
             Self::Status(status_command) => {
-                println!("{:?}", status_command.exec(rest)?);
+                println!("{:?}", status_command.exec::<State>(rest.into())?);
```

Comment:
debugging println?

Reply:
actually not. this is what we print on to user since it's cli